subsys/net/lib/sockets/getaddrinfo.c

```diff
@@ -155,6 +167,35 @@ int z_impl_z_zsock_getaddrinfo_internal(const char *host, const char *service,
 		return getaddrinfo_null_host(port, hints, res);
 	}
 
+#define SIN_ADDR(ptr) (((struct sockaddr_in *)(ptr))->sin_addr)
```

Comment:
You can use here the `net_sin()` func which will do the same.

Reply:
> You can use here the net_sin()

Good hint, fixed, thanks.